docs/dev_guide/documentation.rst

```diff
@@ -10,25 +10,79 @@ Overview
 All code must be documented and we follow these style conventions described here:
 
 * `numpydoc <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_
-* `astropy <https://docs.astropy.org/en/latest/development/docrules.html>`_
 
-We recommend familiarizing yourself with these references.
+We recommend familiarizing yourself with this style.
 
-Differences
------------
+Referring to other code
+-----------------------
 
-The current differences we have are as follows:
+To link to other methods, classes, or modules in sunpy you have to use backticks, for example:
 
-We backtick each type in the documentation strings so that they are interlinked by our documentation builder:
+.. code-block:: rst
 
-.. code-block:: python
+    `sunpy.io.file_tools.read_file`
 
-    """
-    Parameters
-    ----------
-    x : `type`
-       Description of parameter x.
-    """
+generates a link like this: `sunpy.io.file_tools.read_file`.
+
+We use the sphinx setting ``default_role = 'obj'`` so that you don't have to use qualifiers like ``:class:``, ``:func:``, ``:meth:`` (more on this later).
+
+Often, you don't want to show the full package and module name.
+As long as the target is unambiguous you can simply leave them out:
+
+.. code-block:: rst
+
+    `.read_file`
+
+and the link still works: `.read_file`.
```

Comment:
:eyes: it does? What does it reference?!

Reply:
`sunpy.io.read_file`

Reply:
So it finds the (one) matching reference in the current scope? I like that this is a lot shorter, on the other hand though it seems like it's *even more* sphinx magic than typing it all out?

Reply:
Pretty much.